workflow-pacta-cop.Rmd

```diff
@@ -0,0 +1,516 @@
+
+```
+
+Add the portfolio files to the folder `{r} file.path(raw_input_path)`.
+This file should have:
+- investor.name as the peer group name (banks, pensionfund etc)
+- portfolio.name also as the peer group name
+
+```{r echo = FALSE}
+# These run the above project. Will error out if the portfolio file is not provided in the input path
+# For more information on errors that arise, open the workflow-offline.Rmd file
+
+source("2_project_input_analysis.R")
+
+source("3_run_analysis.R")
+
+
+file.copy(file.path(results_path, "Equity_results_portfolio.rda"), file.path(data_path, paste0(project_code,"_peers_equity_results_portfolio.rda")), overwrite = T)
+
+file.copy(file.path(results_path, "Bonds_results_portfolio.rda"), file.path(data_path, paste0(project_code,"_peers_bonds_results_portfolio.rda")), overwrite = T)
+
+```
+
+### Run the QA rmd code
+
+This is a detailed look into the data being used in this analysis. 
+This file is saved in the results folder here: `{r} results_path`
+
+
+```{r echo = FALSE}
+
+ render(file.path("pacta_data_qa.Rmd"),
+         output_dir = results_path,
+         clean = FALSE,
+         quiet = TRUE,
+         params = list(
+            project_code = project_code,
+            project_location_ext = project_location_ext,
+            twodii_internal = twodii_internal
+         )
+         )
+
+
+```
+
+
+### Checking the missing isins
+
+Using the peer group files, it is necessary to check for missing isins. This needs to be obtained and the pacta-data repository updated if necessary. 
+
+Additional missing isins are appended to the file data/isins_missing_bbg_data.csv. This identifies all unique missing isins across all projects.  
+
+This should be analysed using Bloomberg to obtain any new missing isins and these be added to the financial data, before running this again. 
+
+
+### Checking the fund coverage
+
+A coverage summary is provided as an output of 2_project_input_analysis.R. This is a different and more accurate version than what is currently in the QA document. 
+
+```{r echo = FALSE}
+
+# fund_coverage_file
+
+fund_coverage_file <- readRDS(file.path(proc_input_path, "fund_coverage_summary.rda")) %>% 
+  select(investor_name, portfolio_name, isin, value_usd, total_mapped_value_usd, missing_value_usd, funds_in_funds_not_mapped, effective_coverage, fund_data_file_coverage, lipper_data_coverage, lost_coverage_fif)
+
+
+
+```
+
+
+
+
+### Checking the peer group files
+
+Run through the following chunk to check that the files you've created make sense. Make adjustments as necessary. 
+
+```{r echo = FALSE}
+
+# checking the peer group files
+
+# Here is a list of the peer groups being checked for. If your project contains different ones, enter them here. 
+peer_groups <- c("pensionfund","insurance","bank","assetmanager")
+peer_groups <- c("pensionfund", "bank")
+
+# Add an estimated minimum number of investors you have in the study
+number_of_investors = 4
```

Comment:
if this is something you would expect an analyst to change according to their specific situation, as with the user specific file paths I would prefer this to be up front/top and very clearly marked as such... otherwise I would expect users to not notice this